style.css

```diff
@@ -0,0 +1,174 @@
+.navigation-bar{
+    gap:60%;
+}
+
+body{
+    background: rgb(224, 220, 220);
+}
+.categories{
+    
+    text-align: center;
+    
+}
+
+.categories img{
+    padding: 0;
+}
```

Comment:
You can use a code formatter like Prettier to format the code for consistent spacing, will make your code cleaner

Reply:
Okay siddhi will use that.... Thank u